packages/specs/pages/further-reading.mdx

```diff
@@ -6,5 +6,5 @@ Thanks for reading! You can find further information about the project below:
 - [Namada source code](https://github.com/anoma/namada)
 - [Namada community links](https://namada.net/community/docs)
 - [Namada blog](https://blog.namada.net)
-- [Namada Docs](https://docs.namada.net/)
-- [Namada Twitter](https://twitter.com/namada)
+- [Namada docs](https://docs.namada.net/)
+- [Namada Twitter](https://twitter.com/namada)
```

Comment:
Or should we call it X now?

Reply:
No, just because Elon Musk renames something doesn't mean we have to